src/main/java/seedu/planner/model/util/IndexUtil.java

```diff
@@ -75,4 +77,27 @@ public static String convertIndexToYearAndSemester(int index) {
 
         return String.valueOf(year) + String.valueOf(semester);
     }
+
+    public static int[] getIndicesFromYear(int year) {
+        int[] indices = new int[2];
+
+        final int indexZero = 0;
```

Comment:
I actually meant something like `FIRST_SEMESTER_INDEX` and `SECOND_SEMESTER_INDEX` haha. Cos I didn't really understand what 0 and 1 were at first glance